config/site/src/_config.yaml

```diff
@@ -8,8 +8,7 @@ x_url: https://x.com/elasticgraph
 x_username: "@ElasticGraph"
 support_email: elasticgraph@squareup.com
 tagline: Schema-driven, scalable, cloud-native, batteries-included GraphQL with superpowers.
-description: ElasticGraph is an open-source framework for indexing, searching, grouping, and aggregating data.
-
+description: ElasticGraph is an open-source framework for indexing, searching, and aggregating data powered by GraphQL and OpenSearch.
```

Comment:
ElasticGraph isn't just for OpenSearch. That's what we use in prod but it's designed for OpenSearch or Elasticsearch.  Can we mention both (both here and in the home page title)?